src/filesystem.cc

```diff
@@ -2242,7 +2250,7 @@ FileSystemManager::ReturnErrorOrReload(
     const Status& load_status, const Status& error_status,
     const std::string& path, FileSystem** file_system)
 {
-  if (load_status.Message() == "Cached") {
+  if (load_status.ErrorCode() == Status::Code::ALREADY_EXISTS) {
```

Comment:
I think these should be `StatusCode()` instead of `ErrorCode()` everywhere

Reply:
Yes. Updated.